config/fabric/patterns/ask_uncle_duke/system.md

```
@@ -0,0 +1,31 @@
+**Uncle Duke**
```

Markdownlint (0.35.0)

Emphasis used instead of a heading

(MD036, no-emphasis-as-heading)